pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,65 +1,125 @@
+> To find the current version of the [dYdX foundation](https://www.dydx.foundation/) mainnet, see the recommended protocol version on [mintscan.io](https://www.mintscan.io/dydx/parameters).
+
+**Option 1**: Find and download that protocol binary from the [v4 Chain Releases](https://github.com/dydxprotocol/v4-chain/releases/) page.
+> For example, for protocol version 5.0.5 on an AMD system, download `dydxprotocold-v5.0.5-linux-amd64.tar.gz`.
 
-2. For example, as of `04/26/2024`, this was the correct binary to use:
-![dYdX Protocol Binary](../../artifacts/how_to_set_up_full_node_binary_download.png)
-3. Download, extract, and rename the binary to `dydxprotocold`.  Move it to a directory in your `$PATH`.  Now, initialize the dydx home  directory (create the directory first if it doesn’t exist).
+**Option 2**: Download the binary with `curl`, replacing the version numbers and architecture of the package as needed:
 ```bash
-CHAIN_ID=dydx-testnet-4 # or dydx-mainnet-1 for the deployment by DYDX token holders
-NODE_NAME=mydydxfullnode
-dydxprotocold init --chain-id=$CHAIN_ID $NODE_NAME
+# curl example for protocol version 5.0.5 on AMD64 architecture
+curl -L -O https://github.com/dydxprotocol/v4-chain/releases/download/protocol/v5.0.5/dydxprotocold-v5.0.5-linux-amd64.tar.gz
+```
+
+### Step 5: Move `dydxprotocold` to your Cosmovisor `/genesis` directory
+After you download the binary, moving `dydxprotocold` into your Cosmovisor data directory allows you to use Cosmovisor for no-downtime binary upgrades. To extract, rename, and move the file to your Cosmovisor data directory, run the following commands:
 
-by default, the dydx home directory is created in $HOME/.dydxprotocol
+```bash
+# Example for AMD64 architecture
+sudo tar -xzvf dydxprotocold-v5.0.5-linux-amd64.tar.gz # Extract the file
+sudo mv ./build/dydxprotocold-v5.0.5-linux-amd64 ./.dydxprotocol/cosmovisor/genesis/bin/dydxprotocold # Move the file to /.dydxprotocol and rename it
+rm dydxprotocold-v5.0.5-linux-amd64.tar.gz # Delete the installer package
+rm -rf build # Delete the now-empty /build directory
 ```
 
-### Step 5: Create Cosmovisor Directories and Move Binaries
+Add the `dydxprotocold` directory to your system `$PATH`:
+```bash
+echo 'export PATH=$PATH:$HOME/.dydxprotocol/cosmovisor/genesis/bin' >> $HOME/.bashrc # Write to your .bashrc profile
+```
 
+### Step 6: Initialize your node
+To initialize your node, provide the ID of the chain to which you want to connect and create a name for your node. The dYdX home directory is created in `$HOME/.dydxprotocol` by default. Replace the example values `dydx-testnet-0` and `my-node` with your own and run the following command:
 ```bash
-mkdir -p $HOME/.dydxprotocol/cosmovisor/genesis/bin
-mkdir -p $HOME/.dydxprotocol/cosmovisor/upgrades
-mv dydxprotocold $HOME/.dydxprotocol/cosmovisor/genesis/bin/
+dydxprotocold init --chain-id=dydx-testnet-0 my-node
```

Comment:
given that the seed uses prod variables, should we use `dydx-mainnet-1` as the chain-id?

overall, let's be consistent in using the same deployment variables